earthpy/tests/test_plot.py

```diff
@@ -24,87 +24,3 @@ def test_colorbar_raises_value_error():
     with pytest.raises(AttributeError, match="requires a matplotlib"):
         ep.colorbar(list())
     plt.close()
-
-
-""" Hist tests """
```

Comment:
i moved all of the hist tests from this file into a `test_hist` file where it should be!